### DISCUSSION TITLE
Freezing issue when clicking a link, cannot be reproduce consistently

### DISCUSSION THREAD

**Discussion by michaelketzer** [vercel/next.js#10779] [Help] [answered]
Hey,

I'm pretty new to React & Nextjs and just started my second project with it.
I have sometimes the issue that the page is freezing for some seconds when i'm clicking a link to another page.
As I cannot reproduce it consistently and happens rarely, I have not paid attention on this topic yet. Maybe someone can relate the issue and knows what is causing it, i was going through the issue list, but could not find any connection.
I am using the `next-redux-wrapper` which is might not directly visible as it is used in `_app.js`.

The current website can be found here: https://shokz.grief.dev/
And the source code is located here: https://gitlab.com/shokz.tv/web-v2/app

Any help is really apperciated.

Thanks

**Comment by timneutkens** [comment, marked as answer]
It's probably this: https://github.com/zeit/next-plugins/issues/282

You'll want to upgrade to built-in CSS support which doesn't have this issue: https://nextjs.org/blog/next-9-2

**Reply by michaelketzer** [reply]
Thanks a lot. 
I will give it a try :)
